packages/opentelemetry-core/src/baggage/constants.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const KEY_PAIR_SEPARATOR = '=';
```

Comment:
Can you prefix these with `BAGGAGE_` otherwise you have `core.MAX_NAME_VALUE_PAIRS` which could apply to baggage or tracestate or many other things.

Reply:
Even though they weren't exported and i exported utils methods within a `baggageUtils` namespace, i still renamed them because we're never too safe :)